vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -42,7 +43,7 @@ std::string ConvertTypeToString(const ledger::RewardsType type) {
 }
 
 bool HasTokenExpired(const ledger::UnblindedToken& token) {
-  const uint64_t now = static_cast<uint64_t>(base::Time::Now().ToDoubleT());
+  const auto now = braveledger_time_util::GetCurrentTimeStamp();
```

Comment:
I've seen in other reviews a preference for using explicit type names for simple types such as `uint64_t`. I'm good with `auto` here, but I was just curious if I should prefer explicit types in my PRs.

Reply:
yes simple types prefers explicit types